crio/setup.py

```diff
@@ -49,7 +49,7 @@
     ],
 
     # The package we're going to ship
-    packages=['datadog_checks', 'datadog_checks.crio'],
```

Comment:
Was this made from the ddev create command? Does that need updating?

Reply:
Already updated and released a while back